app/src/main/java/com/example/cattlelog/model/dao/CattleDao.kt

```diff
@@ -14,6 +14,8 @@ interface CattleDao {
     fun getCattleWithTagNumber(tagNumber: Int): Cattle?
 //    fun getCattleWithTagNumber(tagNumber: Int): List<Cattle>?
 
+    @Query("SELECT * FROM cattle WHERE TagNumber = :tagNumber AND BirthDate = :birthDate")
+    fun getUniqueHerdMember(tagNumber: Int, birthDate: String): Cattle?
```

Comment:
Nice, I like the renamed function. Much more readable 👍 

This also makes sense since I forgot we have a composite primary key.